cime_config/machines/cmake_macros/gnu_docker.cmake

```diff
@@ -0,0 +1,68 @@
+set(AR "/opt/conda/bin/x86_64-conda-linux-gnu-ar")
```

Comment:
This shouldn't hold up the upstream merge, but I'm making a note to myself and @elynnwu that we should add some comments to this file now that it will also be in the general E3SM code base.  I could easily see some confusion amongst users as to what the docker cmake file is for and why it only works on two machines for now.

Reply:
What is this docker cmake for?  We already have docker capability in E3SM but its under the "singularity" machine.